casper-application-domain/src/main/kotlin/hs/kr/entrydsm/domain/evaluator/policies/EvaluationPolicy.kt

```
@@ -0,0 +1,333 @@
+     * @param node 검증할 AST 노드
+     * @param context 평가 컨텍스트
+     * @return 평가 가능하면 true
+     */
+    fun canEvaluate(node: ASTNode, context: EvaluationContext): Boolean {
+        return try {
+            validateDepth(node, context.maxDepth) &&
+            validateNodeCount(node, DEFAULT_MAX_NODES) &&
+            validateFunctions(node) &&
+            validateOperators(node) &&
+            validateVariables(node, context)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)